models/stg_twilio.yml

```diff
@@ -285,9 +285,9 @@ models:
     tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
-            - account_id
-            - message_id
-            - messaging_service_id
+            - account_id 
+            - message_id 
+            - messaging_service_id 
```

Comment:
Same not null test comment

Reply:
added not null tests to each of these fields, and unique test to message_id

Reply:
update - removed the not null test from `messaging_service_id`. Checking the data, looks like you _don't_ have to be using that field, so it _can_ be null

Reply:
Good call! Is there a reason though `message_id` has it's own unique test as well? I feel that defeats the purpose of the unique combo test.